src/queries/site_state_query.rs

```diff
@@ -2,9 +2,9 @@ use lemmy_api_common::site::GetSiteResponse;
 use leptos::*;
 use leptos_query::{use_query, QueryOptions, QueryResult, RefetchFn, ResourceOption};
 
-#[server(GetSiteResource, "serverfn", "GetJson")]
+#[server(GetSiteResource, "/serverfn", "GetJson")]
```

Comment:
The slash isn't needed.

Reply:
without the slash this gets interpreted as relative (try it on the PostActivity page) so we need the slash